packages/cashier/src/pages/payment-agent/payment-agent-deposit-details/payment-agent-deposit-details.tsx

```diff
@@ -10,80 +10,100 @@ type TPaymentAgentDepositDetails = {
     payment_agent: TPaymentAgent;
 };
 
-const PaymentAgentDepositDetails = ({ payment_agent }: TPaymentAgentDepositDetails) => {
-    const payment_agent_phones = toJS(payment_agent.phone_numbers);
+const PaymentAgentPhonesDetails = (props: {
```

Comment:
@amir-deriv let's destructure props for all components in this file except for the parent (PaymentAgentDepositDetails) and declare types for their props at the top of the file.